m4_sessions/1808-inning/mental_health_in_tech.md

```diff
@@ -1,32 +1,30 @@
 ## Group Members:
-
-
+Austin Wiedenman
+Gavin Love
+Jamison Ordway
+Steve Schwedt
 
 ## Name of Session: 
+Mental Health in Tech Industry
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+1. Effects of Tech Industry culture on mental health conditions, ways to prevent burnout, and how we can support each other.
 
 ## Session Outcomes 
 
-(What will students learn/leave with?): 
+1. An understanding of how issues of mental health are addressed, or not addressed, in the industy.
+2. Ways to find support and how they can support each other.
 
 ## Session Outline 
 
-(Please include who is facilitating and how much time you are going to spend in each section):
-
-## Recommended Trajectory 
-
-     Best practices:  No section should be longer than 20 minutes
-     Create a session that YOU would want to attend
-     Share resources in your slack channel
-     All sessions NEED to have a closing activity, reflection, discussion
-    
-## Introductons 
-## Intro to new material
-## Small Group Activity
-## Closing Discussion
-
-
-Submit a PR at the end of this session and tag @emhickmann
+**(15 min)** Introduction(students)why they chose this session?/ how they implement self care?/ or other?, including presenters personal stories about their experiences here at Turing
+**(10 min)** video
+**(15 min)** small groups, reading differetn articles and discussions as groups.
```

Comment:
@emhickmann If we can get the list of attendees before starting the session, we will have pre-assigned groups in slack so that we can send each group different articles.